libretro/libretro.cpp

```diff
@@ -823,15 +823,6 @@ static void check_variables(CoreParameter &coreParam)
          g_Config.bHardwareTransform = true;
    }
 
-   var.key = "ppsspp_software_skinning";
```

Comment:
Hm, this doesn't actually remove the option in the other file I think?

-[Unknown]